docs/DeveloperGuide.md

```diff
@@ -28,7 +28,7 @@ Refer to the following section for how to **[set up and get started](#setting-up
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Setting up, getting started**
-
+[DeveloperGuide.md](DeveloperGuide.md)
```

Comment:
Is this meant to be in here? If so perhaps consider writing a sentence instead of leaving it as a link, its very jarring and seems completely out of place. Also it just links back to this document?

Reply:
It might have been there due to merging the master branch to this branch, I'll remove it. Thanks for pointing it out.